packages/workflow/src/LoggerProxy.ts

```diff
@@ -0,0 +1,45 @@
+import { 
+	ILogger, 
+	LogTypes,
+} from './Interfaces';
+
+
+let instance: ILogger | undefined;
+
+export function init(logger: ILogger) {
+	instance = logger;
+}
+
+export function getInstance(): ILogger {
+	if (instance === undefined) {
+		throw new Error('Logger not initialized');
+	}
+
+	return instance;
+}
+
+export function log(type: LogTypes, message: string, meta: object = {}) {
+	getInstance().log(type, message, meta);
+}
+
+// Convenience methods below
+
+export function debug(message: string, meta: object = {}) {
+	getInstance().log('debug', message, meta);
```

Comment:
Why not call it like this? Same for the other convenience methods.

```suggestion
	getLogger().debug(message, meta);
```

Reply:
I like the idea of calling the convenience methods statically. Changed the code so it looks like this:

```
import {
    LoggerProxy as Logger,
}  from 'n8n-workflow';

...

LoggerProxy.log(...);
LoggerProxy.error(...);
```